tests/test_integrated_channels/test_integrated_channel/test_exporters/test_content_metadata.py

```diff
@@ -52,34 +54,46 @@ def test_content_exporter_export(self, mock_get_content_metadata):
             FAKE_UUIDS[3],
         ])
 
-    @mock.patch("enterprise.api_client.enterprise_catalog.EnterpriseCatalogApiClient.get_content_metadata")
-    def test_export_with_catalogs_to_transmit(self, mock_get_content_metadata):
+    @mock.patch('integrated_channels.integrated_channel.exporters.content_metadata.EnterpriseCatalogApiClient')
+    def test_export_with_catalogs_to_transmit(self, mock_ent_catalog_api):
```

Comment:
Again, see comments above about making sure other calls to get_content_metadata dont violate existing API contracts. I would leave this test alone and make sure it passes AS-IS for backwards compatibility- your 'test_export_fetches_content_only_when_update_needed' covers the new updates.

Reply:
the issue is that the exporter has changed how it retrieves catalogs, and there's a new API call to `get_enterprise_catalog`, so regardless I have to account for the new mock needed. This is also a test on how the `exporter`'s `export()` method calls the API, not necessarily testing the API itself. I think as such the changes to this test are ok, no?